backend/src/infrastructure/steam.client.js

```diff
@@ -40,7 +40,9 @@ export class SteamClient {
       const url = `https://store.steampowered.com/app/${id}`;
 
       return (await this.#httpClient.get(url)).data;
-    } catch (err) {}
+    } catch (err) {
```

Comment:
Lets stop doing this and introduce propper error handling using error classes. 

Please finish this PR as is and create a new issue for error handling introduction and add a todo with comment here and everywhere this method is used. Same for the method below or any other steam client method where we expect errors.

In the issue add a bulletponit to do some research on how to do this in NodeJS correcty.